test/modifyPluginSpec.js

```diff
@@ -85,6 +85,30 @@ describe('Modify Plugins', function () {
 
       assert.isTrue(this.routeBroadcaster.peersByLedger['eur-ledger-2.']['mark'])
     })
+
+    it('should override the plugin.getInfo function with overrideInfo data', function * () {
+      yield this.app.addPlugin('eur-ledger-2.', {
+        currency: 'EUR',
+        plugin: 'ilp-plugin-mock',
+        options: {
+          prefix: 'eur-ledger-2.'
+        },
+        overrideInfo: {
+          minBalance: '-10',
+          maxBalance: '10000',
+          prefix: 'test.other.prefix.',
+          currencyCode: 'XYZ',
+          currencyScale: 0
+        }
+      })
+
+      const info = this.ledgers._core.clients['eur-ledger-2.'].getPlugin().getInfo()
+      assert.equal(info.minBalance, '-10')
+      assert.equal(info.maxBalance, '10000')
+      assert.equal(info.prefix, 'test.other.prefix.')
+      assert.equal(info.currencyCode, 'XYZ')
+      assert.equal(info.currencyScale, 0)
```

Comment:
maybe avoid the double mention of each value here, maybe using [`assert.include(info, overrideInfo)`](http://chaijs.com/api/assert/#method_include) or even just `assert.deepEqual(info, overrideInfo)` probably (if you don't want `info` to have any fields that `overrideInfo` doesn't have)